mp/src/game/client/neo/ui/neo_hud_round_state.cpp

```diff
@@ -270,113 +288,179 @@ void CNEOHud_RoundState::DrawNeoHudElement()
+									m_rectRightTeamTotalLogo.y0,
+									m_rectRightTeamTotalLogo.x1,
+									m_rectRightTeamTotalLogo.y1);
+
+		// Draw score
+		surface()->GetTextSize(m_hOCRFont, m_wszLeftTeamScore, fontWidth, fontHeight);
+		surface()->DrawSetTextFont(m_hOCRFont);
+		surface()->DrawSetTextPos(m_posLeftTeamScore.x - (fontWidth / 2), m_posLeftTeamScore.y);
+		surface()->DrawSetTextColor(leftTeamInfo.color);
+		surface()->DrawPrintText(m_wszLeftTeamScore, 2);
+
+		surface()->GetTextSize(m_hOCRFont, m_wszRightTeamScore, fontWidth, fontHeight);
+		surface()->DrawSetTextPos(m_posRightTeamScore.x - (fontWidth / 2), m_posRightTeamScore.y);
+		surface()->DrawSetTextColor(rightTeamInfo.color);
+		surface()->DrawPrintText(m_wszRightTeamScore, 2);
+	
+		// Draw total players alive
+		surface()->DrawSetTextFont(m_hOCRSmallFont);
+		surface()->GetTextSize(m_hOCRSmallFont, m_wszPlayersAliveUnicode, fontWidth, fontHeight);
+		surface()->DrawSetTextColor(COLOR_WHITE);
+		surface()->DrawSetTextPos(m_iXpos - (fontWidth / 2), m_ilogoSize);
+		surface()->DrawPrintText(m_wszPlayersAliveUnicode, 9);
+
+	}
+	else
+	{
+		if (g_PR)
+		{
+			// Draw members of players squad in an old style list
+			const int localPlayerTeam = GetLocalPlayerTeam();
+			const int localPlayerIndex = GetLocalPlayerIndex();
+			const bool localPlayerSpec = !(localPlayerTeam == TEAM_JINRAI || localPlayerTeam == TEAM_NSF);
+
+			if (!localPlayerSpec && g_PR->GetStar(localPlayerIndex) != 0)
 			{
-				const int xOffset = (m_iRightOffset + 4) + (rightCount * m_ilogoSize) + (rightCount * 2);
-				DrawPlayer(i, rightCount, rightTeamInfo, xOffset, localPlayerSpec);
-				rightCount++;
+				const int leftTeam = localPlayerSpec ? TEAM_JINRAI : localPlayerTeam;
+				const auto leftTeamInfo = m_teamLogoColors[leftTeam];
+				int leftCount = 0;
+
+				for (int i = 0; i < (MAX_PLAYERS + 1); i++)
+				{
+					if (i == localPlayerIndex)
+					{
+						continue;
+					}
+
+					if (!g_PR->IsConnected(i))
+					{
+						continue;
+					}
+
+					const int playerTeam = g_PR->GetTeam(i);
+					if (playerTeam != leftTeam)
+					{
+						continue;
+					}
+
+					const bool isSameSquad = g_PR->GetStar(i) == g_PR->GetStar(localPlayerIndex);
+					if (!isSameSquad)
+					{
+						continue;
+					}
+
+					// Draw player
+					static constexpr int SQUAD_MATE_TEXT_LENGTH = 62; // 31 characters in name without end character max plus 3 in short rank name plus 7 max in class name plus 3 max in health plus other characters
+					char squadMateText[SQUAD_MATE_TEXT_LENGTH];
+					wchar_t wSquadMateText[SQUAD_MATE_TEXT_LENGTH];
+					const char* squadMateRankName = GetRankName(g_PR->GetXP(i), true);
+					const char* squadMateClass = GetNeoClassName(g_PR->GetClass(i));
+					const int squadMateHealth = g_PR->IsAlive( i ) ? g_PR->GetHealth( i ) : 0;
+					V_snprintf(squadMateText, SQUAD_MATE_TEXT_LENGTH, "%s %s  [%s]  Integrity %i\0", g_PR->GetPlayerName( i ), squadMateRankName, squadMateClass, squadMateHealth);
```

Comment:
There's no point of the `\0` at the end when it's already applied for C strings. But not going to prevent the PR just for this.